### PR TITLE
FIX : update tunnel-agent to 0.6.0

### PR DESCRIPTION
**Pull request by paolo-tanium**
## Pre-Submission Checklist

- [x] Opened an issue discussing these changes before opening the PR

## Issues

Closes:

* Issue #1654 

# Changes

Upgrade tunnel-agent to 0.6.0. The previous version (0.4.3) was being reported by nsp as having a vulnerability, which was addressed in 0.6.0.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
   "name": "restify",
   "homepage": "http://restifyjs.com",
   "description": "REST framework",
-  "version": "4.3.3",
+  "version": "4.3.4",
```
Comment (hekike) [inline package.json]: Please remove the version, we use a semi-automated publishing process which will handle it.

Comment (paolo-tanium) [reply package.json]: Fixed in https://github.com/restify/node-restify/pull/1656/commits/dd8f3dd8533abc6bdcfcb79e03100ed0bfb04c47

**Comment by distracteddev** [comment]
Thanks for updating old versions! <3 When can we expect this to be published to the 4.x line?

**Comment by DonutEspresso** [comment]
v4.3.4 is out! 🎉

**Pull request merged**